### PR TITLE
Markdownlint Bugfix

### LINKED ISSUE #1560: [BUG] Markdown Lint Configuration
## Description
There is a need to lint all of the markdown files in the repository and eliminate all of the exclusions currently in place. However, it is not clear where the markdown linting exclusions are set.

## Expected Behavior
All markdown files in the repository should be linted, including those that are currently excluded from the linting process.

## Actual Behavior
Markdown files in the repository are being linted, but there are exclusions in place that prevent some files from being checked. It is unclear where these exclusions are defined or how to modify them.

### PR DESCRIPTION
**Pull request by eduardomr**
Closes #1560

### CODE REVIEW

**Comment by BenSturmfels** [comment]
Thanks for the PR @eduardomr, @joseluis-rt, @klyssmannoliveira. The `.github/workflows/pr.yml` was just calling the command `markdownlint` from `package.json`, so I'd suggest making the change there instead. That way you'll also fix the `npx markdownlint` command when other developers run it from the command-line (not just in CI).

**Comment by eduardomr** [comment]
Thanks for the feedback @BenSturmfels! We'll do the changes.

**Comment by BenSturmfels** [comment]
> Thanks for the feedback @BenSturmfels! We'll do the changes.

@eduardomr it would be great to get these changes transferred to `package.json` when you have a moment.

**Comment by BenSturmfels** [comment]
@eduardomr that's great. Now please revert the change to `.github/workflows/pr.yml` and this will be good to go.

File: .github/workflows/pr.yml
Comment (BenSturmfels) [inline .github/workflows/pr.yml:1]: @eduardomr getting close, but please remove the changes made to this file (.github/workflows/pr.yml).

**Comment by wwahammy** [comment]
Thanks @eduardomr and @BenSturmfels, I've gone ahead and corrected the pr.yml file and we should be able to merge the PR. Thanks for all of the help everyone!

**Pull request merged**